dash-renderer/src/actions/index.js

```diff
@@ -544,12 +571,29 @@ function handleClientside(clientside_function, payload) {
     let returnValue;
 
     try {
+        // setup callback context
+        const input_dict = inputsToDict(inputs);
+        dc.callback_context = {};
+        if (payload.changedPropIds.length === 0) {
+            dc.callback_context.triggered = [{prop_id: '.', value: null}];
```

Comment:
I don't think we want this dummy value on the client side. On the server it's there for backward compatibility as versions before 1.11 would always have at least one entry in `triggered` (and in fact almost always had exactly one entry) so people were in the habit of just looking at `triggered[0]["prop_id"]`. In JS there's no such history to work around. Also in Python we have the flexibility to make this length-1 array [look falsy](https://github.com/plotly/dash/blob/439e13b673e8e8f3d4223e905d44c08c9f11eef2/dash/_callback_context.py#L21) which we can't do in JS.

Reply:
Sounds good. I can just remove the `if` part of that conditional.